azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/AppServiceResourceManager.java

```diff
@@ -52,7 +52,7 @@ protected AppServiceResourceManager(@Nonnull AppServiceManager remote, @Nonnull
     @Nonnull
     @Override
     public List<AzResourceModule<?, AppServiceResourceManager, ?>> getSubModules() {
-        return Arrays.asList(webAppModule, functionAppModule, planModule);
+        return Collections.singletonList(planModule);
```

Comment:
Please comment why we need to remove webApp and function moudle from `AppServiceResourceManager`, which is quite confusing.